src/redteam/constants.ts

```diff
@@ -363,7 +363,7 @@ export const ALIASED_PLUGIN_MAPPINGS: Record<
   'owasp:llm': OWASP_LLM_TOP_10_MAPPING,
 };
 
-export const DEFAULT_STRATEGIES = ['jailbreak', 'prompt-injection'] as const;
+export const DEFAULT_STRATEGIES = ['jailbreak', 'jailbreak:composite', 'prompt-injection'] as const;
```

Comment:
we should remove 'prompt-injection'

Reply:
is composite going to be better?

Reply:
Much! I want to get some stats on our website on a strategies page: https://github.com/promptfoo/promptfoo/compare/main...docs/strategy-overview